tablecodec/tablecodec.go

```diff
@@ -971,8 +971,8 @@ func GenIndexKey(sc *stmtctx.StatementContext, tblInfo *model.TableInfo, idxInfo
 }
 
 // GenIndexValue creates encoded index value and returns the result
-func GenIndexValue(sc *stmtctx.StatementContext, tblInfo *model.TableInfo, idxInfo *model.IndexInfo,
-	containNonBinaryString bool, distinct bool, untouched bool, indexedValues []types.Datum, h kv.Handle) ([]byte, error) {
+func GenIndexValue(sc *stmtctx.StatementContext, tblInfo *model.TableInfo, idxInfo *model.IndexInfo, containNonBinaryString bool,
```

Comment:
If there are multiple reference of this function in other packages, it's better to introduce a new function such as `GenGlobalIndexValue` instead of changing the API

Reply:
This API does not change, I just split a long single line into two.

Reply:
You add a `partitionID int64`, right?